tests/auth.go

```diff
@@ -45,14 +45,17 @@ func ClientConfigMA() (ma.Multiaddr, string) {
 	if err != nil {
 		panic(err)
 	}
-	home, err := os.UserHomeDir()
-	if err != nil {
-		panic(err)
-	}
-	path := filepath.Join(home, ".lotus")
-	authToken, err := GetLotusToken(path)
-	if err != nil {
-		panic(err)
+	authToken, ok := os.LookupEnv("TEXTILE_LOTUS_TOKEN")
```

Comment:
Also looks at an env var, used in docker-compose. 
We might have a more clear policy to name env vars. For the moment, choose this.